ui/src/components/navigation/Sidebar.tsx

```diff
@@ -0,0 +1,77 @@
+import { FC, ReactElement, useState } from 'react';
+import {
+  Box,
+  Drawer,
+  List,
+  Divider,
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+} from '@mui/material';
+import HomeIcon from '@mui/icons-material/Home';
+import MusicNoteIcon from '@mui/icons-material/MusicNote';
+import QueueMusicIcon from '@mui/icons-material/QueueMusic';
+import SupervisedUserCircleIcon from '@mui/icons-material/SupervisedUserCircle';
+import TextSnippetIcon from '@mui/icons-material/TextSnippet';
+
+const drawerWidth = 240;
+
+const SideBar: FC = (): ReactElement => {
+  const [selectedItem, setSelectedItem] = useState<string | null>(null);
+
+  const handleClick = (text: string) => {
+    setSelectedItem(text);
+  };
+
+  return (
+    <Box sx={{ display: 'flex' }}>
+      <Drawer
+        sx={{
+          width: drawerWidth,
+          flexShrink: 0,
+          '& .MuiDrawer-paper': {
+            width: drawerWidth,
+            boxSizing: 'border-box',
+          },
+        }}
+        variant="permanent"
+        anchor="left"
+      >
+        <Divider />
+        <List sx={{ marginTop: '4em' }}>
+          {[
```

Comment:
can this array of icons and text be in another variable? it feels very full and complex to read this code

and it's also the same one as the SidebarMobile so you might as well combine it together so you just need to change in one place

Reply:
good catch, addressed. removed sidebarmobile and declared list outside